src/components/map-marker/map-marker.tsx

```diff
@@ -0,0 +1,118 @@
+import { Component, Element, Event, EventEmitter, Prop } from "@stencil/core";
+import { IComponent } from "../common/interfaces";
+import {
+  icon,
+  marker
+  // tslint:disable-next-line:no-submodule-imports
+} from "leaflet/dist/leaflet-src.esm";
+
+@Component({
+  shadow: false,
+  styleUrl: "map-marker.scss",
+  tag: "gx-map-marker"
+})
+export class MapMarker implements IComponent {
+  @Element() element: HTMLElement;
+
+  private markerInstance: any;
+
+  /**
+   * The coordinates where the marker will appear in the map.
+   *
+   */
+  @Prop({ mutable: true })
+  coords = "0, 0";
+
+  /**
+   * The src of marker img. It can be an URL or a file path.
+   *
+   */
+  @Prop({ mutable: true })
+  iconSrc = "https://unpkg.com/leaflet@1.3.4/dist/images/marker-icon.png";
+
+  /**
+   * The size that the marker img will have.
+   *
+   */
+  @Prop({ mutable: true })
+  iconSize = "25, 41";
```

Comment:
I think this one shouldn't be mutable.

Could this be done via CSS instead of using an attribute?

Reply:
I do not think so. When u build the marker obj, those values r used by [Leaflet](https://leafletjs.com) obj for set style to the drawn marker. (Look at the img)
![image](https://user-images.githubusercontent.com/33690842/47656888-cd2d8c80-db6e-11e8-8d5c-442d586ab2e8.png)


Perhaps u have another solution or I did not understood. But I think using CSS in the element `gx-map-marker` would not make change on generated `<img></img>` element by [Leaflet](https://leafletjs.com)

Reply:
Ok, I agree. I think the best for now would be to leave the icon size out of the scope of this first version, until we figure out a better way of setting it up that's both familiar and easy to use.
I think that for now we can settle with a fixed detault icon size.